src/style.css

```
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+}
+
+.star-block {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(1),
```

```suggestion
.stars--1 .stars__star:nth-child(-n+1),
```
Replace to make code the same style